### PR TITLE
Update Baden-Württemberg

### PR DESCRIPTION
**Pull request by NxrVincent**
added DB Regio AG Baden-Württemberg:
RE 7 (RK - RB)
RB 28 (RF - XFMV)
RB 27 (RF - RB)
RB 26 (RF - RO)

updated "rvf-sbahn":
S1, S10 & S11 to the right colors

source: https://www.bwegt.de/fileadmin/assets/www.bwegt.de/0000_v2/c_PDFs/Karten/bwegt_Liniennetzplan_Regionalverkehr_BW_08.12.2023_barrierefrei.pdf

### CODE REVIEW

**Review by marhei — CHANGES_REQUESTED** [review]
Thank you for your contribution, please add yourself to the sources.json.

File: line-colors.csv [hunk 1]
```diff
@@ -94,6 +94,12 @@ db-regio-bayern,RE 73,db-regio-ag-bayern,re-73,#e5a05c,#ffffff,,rectangle
 db-regio-bayern,RE 75,db-regio-ag-bayern,re-75,#e50000,#ffffff,,rectangle
 db-regio-bayern,RE 76,db-regio-ag-bayern,re-76,#006629,#ffffff,,rectangle
 db-regio-bayern,RE 79,db-regio-ag-bayern,re-79,#800080,#ffffff,,rectangle
+db-regio-ag-baden-wurttemberg,RE7,db-regio-ag-baden-wurttemberg,re-7,#ED6E19,#ffffff,,rectangle
```
Comment (marhei) [inline line-colors.csv:97]: "Uncaught Error: backgroundColor is not lowercase in row 97"

**Comment by NxrVincent** [comment]
Now it's updated

File: line-colors.csv [hunk 2]
```diff
@@ -94,6 +94,12 @@ db-regio-bayern,RE 73,db-regio-ag-bayern,re-73,#e5a05c,#ffffff,,rectangle
 db-regio-bayern,RE 75,db-regio-ag-bayern,re-75,#e50000,#ffffff,,rectangle
 db-regio-bayern,RE 76,db-regio-ag-bayern,re-76,#006629,#ffffff,,rectangle
 db-regio-bayern,RE 79,db-regio-ag-bayern,re-79,#800080,#ffffff,,rectangle
+db-regio-ag-baden-wurttemberg,RE7,db-regio-ag-baden-wurttemberg,re-7,#ed6e19,#ffffff,,rectangle
```
Comment (marhei) [inline line-colors.csv:97]: The operators should be in alphabetical order: "db-regio-bayern should be after db-regio-ag-baden-wurttemberg in row 97"

**Review by marhei — APPROVED** [review]
Perfect, thank you!

**Pull request merged**